Tasks/CondaEnvironmentV0/conda.ts

```diff
@@ -16,7 +16,6 @@ interface TaskParameters {
     cleanEnvironment?: boolean
 }
 
-// TODO copied from PythonScript. Find a way to share between tasks.
```

Comment:
We've since learned that the "common" directory is not a good solution because it subverts the dependency system, and I don't think this is a good candidate for inclusion in task-lib since it is somewhat specific to the pattern we have been using for the new CIX tasks.